source/common/common/backoff_strategy.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/backoff_strategy.h"
+
+namespace Envoy {
+
+ExponentialBackOffStrategy::ExponentialBackOffStrategy(const uint64_t initial_interval,
+                                                       const uint64_t max_interval,
+                                                       const double multiplier)
+    : initial_interval_(initial_interval), max_interval_(max_interval), multiplier_(multiplier),
+      current_interval_(0) {
+  ASSERT(multiplier_ >= 1.5);
```

Comment:
Why?

Reply:
do you want it to be more than 2? I thought people can use 1.5 as multiplier as well. that is why I modified this.

Reply:
I would suggest `>= 1.0`; 1.125 would be a valid value, for example.

Reply:
ok. I think we discussed earlier that should be more than 1.0. So would validate against >1.0